Source/WebKit/WebProcess/Extensions/API/Cocoa/WebExtensionAPIPermissionsCocoa.mm

```diff
@@ -165,7 +165,7 @@
 
 bool WebExtensionAPIPermissions::verifyRequestedPermissions(HashSet<String>& permissions, HashSet<Ref<WebExtensionMatchPattern>>& matchPatterns, NSString *callingAPIName, NSString **outExceptionString)
 {
-    auto extension = WebExtension::create(extensionContext().manifest(), @{ });
+    auto extension = WebExtension::create(extensionContext().manifest(), HashMap<String, std::span<const uint8_t>>());
```

Comment:
```suggestion
    auto extension = WebExtension::create(extensionContext().manifest(), { });
```

Reply:
`error: no matching function for call to 'create'`.

Reply:
Can you change `explicit WebExtension(NSDictionary *manifest, const Resources&)` to be `explicit WebExtension(NSDictionary *manifest, const Resources& = { })`, and just remove the empty object at this call site then?